### ISSUE TITLE
TypeScript: Incorrectly reordering access and static modifiers in classes

### ISSUE THREAD

**Issue by prencher** [prettier/prettier#1884]
Input:
```ts
class Foo {
  public static bar = 1;
  protected static fluff = 2;
  private static giraffe = 3;
}
```

Outputs the following invalid TypeScript:
```ts
class Foo {
  static public bar = 1;
  static protected fluff = 2;
  static private giraffe = 3;
}
```

**Comment by vjeux** [comment]
#1883 is the same issue.

**Comment by vjeux** [comment]
I just published 1.4.1 which fixes it!
